### PR TITLE
Coffee system support

### PR DESCRIPTION
**Pull request by aturri**
Added complete support to coffee machines, tested with CM6160 and CVA7845 models.
Fix program type 4 that was reported as cleaning/care program, instead of maintenance.

### CODE REVIEW

File: custom_components/miele/translations/sensor.nb.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
       "normal_operation_mode": "Normalmodus",
       "own_program": "Eget program",
       "automatic_program": "Automatiskt program",
-      "cleaning_care_program": "Rengjøringsprogram"
+      "cleaning_care_program": "Rengjøringsprogram",
+      "maintainance_program": "Maintainance program"
```
Comment (astrandb) [inline custom_components/miele/translations/sensor.nb.json:27]: Spelling should be `maintenance` - in many places and files.

**Review by astrandb — CHANGES_REQUESTED** [review]
Looks good to me.

File: custom_components/miele/translations/sensor.nb.json (hunk 1 above)
Comment (aturri) [reply custom_components/miele/translations/sensor.nb.json]: wooops sorry :)

**Review by astrandb — APPROVED** [review]
Thanks!

**Pull request merged**